### ISSUE TITLE
Program a filter a filter during a search

### ISSUE THREAD

**Issue by Chungaa** [libre-tube/LibreTube#3282]
### Describe your suggested feature

If you type something in the search bar and the videos are displayed.
That one builds in a filter for example: Videos this week or videos this month or videos this year. A kind of filter
As it is with the web version and original Youtube app.
here is a Phtoto

![Screenshot 2023-03-10 161042](https://user-images.githubusercontent.com/109001366/224352139-7248f00a-084c-4427-8c35-15bc96f54adf.png)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
This won't be possible before https://github.com/TeamNewPipe/NewPipeExtractor/pull/904 is merged, so please open a new issue once that PR is merged.
